From e4ee8feed7a2b460bbb202704072c1927015265f Mon Sep 17 00:00:00 2001
From: Adrian Bunk <bunk@debian.org>
Date: Thu, 31 Oct 2019 12:28:48 +0200
Subject: [PATCH] Don't build with -Werror

New compiler versions might output new warnings by default
(see #47 for an example), which makes -Werror problematic
for distributions.

The more specific -Werror= settings are usually not a problem.

The warnings are still shown, but they will no longer break
the build.

Signed-off-by: Adrian Bunk <bunk@debian.org>
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 7bc20f8..7e01bd5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -22,7 +22,6 @@ AX_AM_CFLAGS_ADD([-Werror=format-truncation=1])
 AX_AM_CFLAGS_ADD([-Werror=shift-negative-value])
 AX_AM_CFLAGS_ADD([-Werror=alloca])
 AX_AM_CFLAGS_ADD([-Werror=missing-field-initializers])
-AX_AM_CFLAGS_ADD([-Werror])
 AX_AM_CFLAGS_ADD([-Werror=format-signedness])
 
 AC_SUBST([AM_CFLAGS])
-- 
2.24.0

